agents/agents-ext/src/commonMain/kotlin/ai/koog/agents/ext/agent/AIAgentSubgraphExt.kt

```diff
@@ -143,15 +143,15 @@ public object ProvideVerifiedSubgraphResult : ProvideSubgraphResult<VerifiedSubg
  * - Execution logic that returns the input `StringSubgraphResult` as output without transformation.
  *
  * Features:
- * - The `descriptor` describes the tool as "finish_task_execution" and requires a parameter named "result"
+ * - The `descriptor` describes the tool as "finish_task_execution_string" and requires a parameter named "result"
```

Comment:
Here as well. But TBH I think the entire Features section can be removed, it's too verbose